packages/flutter_driver/lib/src/extension/extension.dart

```diff
@@ -262,6 +263,15 @@ class FlutterDriverExtension {
     }, description: 'widget with text tooltip "${arguments.text}"');
   }
 
+  Finder _createBySemanticLabelFinder(BySemanticLabel arguments) {
+    return find.byElementPredicate((Element element) {
+      final Widget widget = element.widget;
+      if (widget is Semantics)
```

Comment:
What about labels that were introduced via RenderObject.describeSemanticsConfiguration?

Reply:
I would use logic similar to what exists already

https://github.com/flutter/flutter/blob/440ce8fde694f14c2c6b008600182935a7b8c14d/packages/flutter_test/lib/src/widget_tester.dart#L701

Reply:
My attempt at doing this is giving me strange results - when I search by label, I'm getting multiple render objects tellling me their semantic label is the label I want.  However, the semantics tree is correct (?!).

I can't quite use the existing logic there because that assumes you have what you're looking for already and just want the semantics - I want to use semantics to get what I'm looking for.

Reply:
Figured it out - I need to check if the element I'm inspecting is a RenderObjectElement.  This is ready for a look again.